node/rate_limiter.js

```diff
@@ -72,7 +75,18 @@ function RateLimiter(options) {
     }
     var self = this;
 
+    // stats
+    self.statEmitter = options.statEmitter || new StatEmitter();
```

Comment:
let's not add a new stats emitter. Let's do it all through `channel.buildStat()`